maps/away/away_sites.dm

```diff
@@ -15,21 +15,40 @@ ABSTRACT_TYPE(/datum/map_template/ruin/away_site)
 	/// where exoplanet generation with the map value is applied only on marker turfs of the applicable map key.
 	var/list/exoplanet_themes = null
 
-	/// Light level of exoplanet turfs if they're generated on this away site. See code\modules\overmap\exoplanets\decor\_turfs.dm:36
-	var/lightlevel = 0
-	/// Light color of exoplanet turfs if they're generated on this away site. See code\modules\overmap\exoplanets\decor\_turfs.dm:36
-	var/lightcolor = COLOR_WHITE
+	/// Light level of exoplanet turfs if they're generated on this away site.
+	/// Only outside turfs get light, based on the `is_outside` var on areas.
+	/// If list, picks one from the list.
+	/// If zero, ignored, no light.
+	var/exoplanet_lightlevel = 0
+	/// Light color of exoplanet turfs if they're generated on this away site.
+	/// If list, picks one from the list.
+	/// Used only if lightlevel is set to non-zero value.
+	var/exoplanet_lightcolor = COLOR_WHITE
+
 	/// The atmosphere that exoplanet turfs should spawn with.
+	/// If null, ignored, and turfs keep their default or mapped in atmosphere.
+	/// If set, this atmosphere is used. If list, picks one from the list.
+	/// Should be set to a `/datum/gas_mixture` path.
```

Comment:
This is an illicit usage of a typed var, it should always only contain the type it declares or null; if you want it to pick from a list of options, you can pass said options via another var, use a proc to do it, or untype the var

Reply:
changed to be `var/list/datum/gas_mixture/exoplanet_atmosphere` and the comment to say it has to be a list of these paths

Reply:
This is still not an acceptable solution, as it won't be a list after New()

Reply:
yeah I was tired and sick when I did this

added a new var that is a list, and one item is picked from that list and put in the actual var